docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -8,6 +8,10 @@
 
   &.hb-page-width {
     @include hb-page-width;
+    // scss-lint:disable ImportantRule
```

Comment:
I've added this in my PR, all the new green stuff :) it will all be part of the hb-page-width mixin now